rust/stackable-cockpit/src/xfer/mod.rs

```diff
@@ -82,7 +85,9 @@ impl Client {
     }
 
     async fn get_from_local_file(&self, path: &PathBuf) -> Result<String> {
-        fs::read_to_string(path).await.context(ReadLocalFileSnafu)
+        fs::read_to_string(path)
+            .await
+            .context(ReadLocalFileSnafu { file: path })
```

Comment:
note: This can then use the short syntax.

```suggestion
            .context(ReadLocalFileSnafu { path })
```